com.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,99 @@
+id: com.github.slgobinath.SafeEyes
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+command: safeeyes
+
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.DBus" # for tray icon
+  - "--talk-name=org.freedesktop.Notifications" # for 'notification before breaks' plugin
```

Comment:
are you sure? libnotification 0.8.2 uses the portal and doesn't need any permission

Reply:
Tried a couple of times and it didn't work without it. Let me try again though